RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## New in git-machete 3.11.2
+
+- fixed: `git machete` now correctly infers remote for fetching of branch when the branch is found in more than one remote
```

Comment:
Maybe actually:
```suggestion
- fixed: `git machete` now correctly does not infer a remote for fetching of branch when the branch is found in more than one remote
```

Reply:
@PawelLipski Why? The code still infers the remote, just the correct one, i would't change change it like that IMHO

Reply:
Okay, let's leave as is, not insisting